sdk/modelsrepository/Azure.IoT.ModelsRepository/samples/readme.md

```diff
@@ -1,9 +1,11 @@
-# IoT Models Repository Samples
+# IoT Models Repository Client Samples
 
-The Azure IoT Models Repository enables builders to manage and share digital twin models for global consumption. The models are [JSON-LD][json_ld_reference] documents defined using the Digital Twins Definition Language ([DTDL][dtdlv2_reference]).
+The Azure IoT Models Repository Client enables builders to manage and share digital twin models for global consumption. The models are [JSON-LD][json_ld_reference] documents defined using the Digital Twins Definition Language ([DTDL][dtdlv2_reference]).
```

Comment:
This sentence was meant to give a background on the service that the client is for.

Reply:
Looking at the first sentence in this MS doc about the repository https://docs.microsoft.com/en-us/azure/iot-develop/concepts-model-repository , has very similar verbiage without the usage of "Client".